native-stack/views/NativeStackView.tsx

```diff
@@ -46,7 +46,7 @@ export default function NativeStackView({
           <Screen
             key={route.key}
             style={StyleSheet.absoluteFill}
-            gestureEnabled={gestureEnabled}
+            gestureEnabled={Platform.OS === 'android' ? false : gestureEnabled}
```

Comment:
swipe gestures are common on android these days tho. there will be no way to have them on android with this change, yeah?

Reply:
`gestureEnabled` prop on Android handles dismissing screen by clicking hw back button and there are no swipe gestures supported by RNScreens on Android for now.